### PR TITLE
Feature/unit tests coverage

### PR DESCRIPTION
**Pull request by brainthinks**
initial test coverage set up

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -18,10 +18,12 @@ module.exports = {
   clearMocks: true,
 
   // Indicates whether the coverage information should be collected while executing the test
-  // collectCoverage: false,
+  collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  // collectCoverageFrom: undefined,
+  collectCoverageFrom: [
+    "<rootDir>/src/js/conduit/Router.js"
```
Comment (csmcgee) [inline jest.config.js]: @brainthinks have we thought about how another developer would add files as they go along? is this required?

File: package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
     "preversion": "./scripts/version.sh --pre",
     "version": "./scripts/version.sh",
     "postversion": "./scripts/version.sh --post",
-    "test": "jest"
+    "test": "jest && ./test/jest/scripts/post-test.sh"
```
Comment (csmcgee) [inline package.json]: @brainthinks what if I want jest to watch my changes? Can I still use this?

Comment (csmcgee) [reply package.json]: or are we going to use another script command for developer use and use this one only for ci.

Comment (brainthinks) [reply package.json]: is setting up a watcher for testing a thing?

Comment (csmcgee) [reply package.json]: add `test:watch` script for people who want to watch their tests.

Comment (csmcgee) [reply package.json]: add `test:min` or `test:dev` some script in which developers would use that would be FAST for them to get feedback. As code coverage is not something a developer may care about seeing when working on a bug or feature right away.

File: jest.config.js (hunk 1 above)
Comment (brainthinks) [reply jest.config.js]: changed this to be the src/js directory

File: package.json (hunk 2 above)
Comment (brainthinks) [reply package.json]: added

**Comment by brainthinks** [comment]
these changes have been added to STRY0060808

**Pull request closed without merging**